test/tfvc/commands/findworkspace.test.ts

```diff
@@ -0,0 +1,81 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+"use strict";
+
+import { FindWorkspace } from "../../../src/tfvc/commands/findworkspace";
+import { TfvcError } from "../../../src/tfvc/tfvcerror";
+import { IExecutionResult, IWorkspace } from "../../../src/tfvc/interfaces";
+
+var chai = require("chai");
+/* tslint:disable:no-unused-variable */
+var expect = chai.expect;
+/* tslint:enable:no-unused-variable */
+var assert = chai.assert;
+chai.should();
+
+describe("FindWorkspaceCommand", function() {
```

Comment:
Perhaps we should have a prefix like "Tfvc-" on the describe so we can immediately see that it's a part of that extension?